code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -291,11 +278,15 @@
 				var/mob/living/carbon/C = L
 				switch(firer.a_intent)
 					if(INTENT_HELP)
-						C.visible_message("<span class='danger'>[L] is pulled by [H]'s tentacle!</span>","<span class='userdanger'>A tentacle grabs you and pulls you towards [H]!</span>")
+						C.visible_message("<span class='danger'>[L] is pulled to and up by [H]'s tentacle!</span>","<span class='userdanger'>A tentacle grabs you and pulls you up towards [H]!</span>")
```

Comment:
`is pulled to and up by [H]` should probably be `is pulled to their feet towards [H]`

Reply:
done